docs/api-testing/composer/expressions.md

```diff
@@ -0,0 +1,234 @@
+---
+id: expressions
+title: Using Expressions in Test Components
+sidebar_label: Using Expressions
+description: Learn to use expressions in components.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Expressions are fields that reference an item in the test scope. The item can be a variable or an inner value in a data structure, such as a JSON.
+
+## What You'll Need
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+* Familiarity with the [API Testing Composer](/api-testing/composer/).
+
+## Using Expressions
+
+When working with structured data, expression is the path for reaching out a specific element. Most of the time, it's just object dot notation. Most expressions will start with the name of the variable the data is stored in. In this example, we will assume the data has been assigned to a variable named `payload`:
+
+```json
+"data":{
+  "created_time": "1453198835",
+  "images": {
+    "thumbnail": {
+      "width": 150,
+      "url": "https://domain.com/photos/9451802655724661601789699.jpg",
+      "height": 150
+      }
+   },
+  "Total-items": 1
+}
+```
+
+If you want to reach the value of the `created_time` attribute:
+
+```js
+payload.data.created_time
+```
+
+If you want to check the `width` of the images:
+
+```js
+payload.data.images.thumbnail.width
+```
+
+### Special Characters
+
+The `"Total-items"` element is a bit tricky, because the minus sign ( - ) would be misunderstood and treated as a subtraction operation. For this reason, the dot notation would require square brackets:
```

Comment:
```suggestion
The `Total-items` element is a bit tricky, because the minus sign ( - ) would be misunderstood and treated as a subtraction operation. For this reason, the dot notation would require square brackets:
```